firmware/controllers/algo/fuel_math.cpp

```diff
@@ -150,20 +151,28 @@ float getRealMafFuel(float airSpeed, int rpm DECLARE_ENGINE_PARAMETER_SUFFIX) {
 		return 0;
 	}
 
+	//Calculation of 100% VE air mass in g/rev
+	float StandardAirCharge = CONFIG(specs.displacement) / CONFIG(specs.cylindersCount) * 1.2929; //1 cylinder filling at 1.2929g/L
```

Comment:
Might be a bit easier to read this if we move this line down to just before the `float airChargeLoad...` line.

Reply:
The comment? 
//1 cylinder filling at 1.2929g/L

Reply:
Oh you mean the whole line. 
See no problem with that.